.github/workflows/build.yml

```diff
@@ -56,7 +56,6 @@ jobs:
     - name: Test docs and CLI
       run: |
         sphinx-build -n -v -E -W ./docs/rst/manual ./tmp/ert_docs
-        pip uninstall -y PyQt5
```

Comment:
This will need some context

Reply:
ERT GUI depends on PyQT, but the CLI does not.  Test docs and CLI workflow step was added to ensure users don't encounter into QT dependency problems while running ERT as CLI. But this issue delegated ERT's script creation to setuptools' entry_points, which uses package_resources to check target's dependency three before creating its entry_point. This cause the problem because PyQT5 is required dependency and was removed in the test, hence the Error.  Unlike the previous version, PyQT5 is now on PyPI and can be easily be added as a dependency with pip install PyQT5 . Thus we can set PyQT as an extra dependency and users who want to use ERT as a GUI can install ERT with pip install ert[gui]. For now the this test will be removed from workflows, but later in a new issue "Prompt user that runs ERT GUI with proper install" we will make a new test that asserts that ERT CLI can be run without PyQT.

Reply:
Awesome. Thanks!